resource_calendar_rrule/models/resource_calendar.py

```diff
@@ -78,15 +59,12 @@ def _simplified_from_attendances(self):
                 if rule._until and (not stop or stop > rule._until):
                     stop = rule._until
 
-                odd = all(
-                    bool(rule._dtstart.isocalendar()[1] % 2)
-                    for occurrence in rule[:2]
-                )
                 key = float(self.env['ir.config_parameter'].get_param(
                     'resource_calendar_rrule.simplified_afternoon',
                     DEFAULT_AFTERNOON_HOUR,
                 )) > attendance.hour_from and 'morning' or 'afternoon'
 
+                odd = bool(rule._dtstart.isocalendar()[1] % 2)
```

Comment:
@hbrunn I don't quite understand what was meant here, but since in the [previous version](https://github.com/OCA/hr/pull/419/files#diff-a560eff6b0bcdbd3621c082965c7cbdcL81) `occurrence` is not used, I think this line of code is equivalent?

Reply:
it's not. This piece of code is meant to detect if this is an rrule implementing the simplified occurrence only for odd weeks. The original looked at two consecutive occurrences, and if both are odd weeks, we know that this is a rule for odd weeks. Otherwise, there would have been an even week in between

Reply:
so change `rrule._dtstart` to `occurrence`, then the code does what it's meant to be ;-)